engine/c.cc

```diff
@@ -197,6 +241,8 @@ KVDKIterator *KVDKCreateIterator(KVDKEngine *engine, const char *collection,
   } else if (iter_type == HASH) {
     result->rep =
         (engine->rep->NewUnorderedIterator(std::string(collection))).get();
+  } else {
+    return nullptr;
   }
   assert(result->rep != nullptr && "Create Iterator Failed!");
```

Comment:
Should also return nullptr here?

Reply:
It's not up to user to check result->rep. User only needs to check result != nullptr.

Reply:
ok